designsafe/static/scripts/projects/components/manage-project/manage-project.component.js

```diff
@@ -180,13 +182,32 @@ class ManageProjectCtrl {
                 guest.user = "guest" + guest.fname + guest.lname.charAt(0) + i;
             });
             projectData.nhTypes = this.form.nhTypes.filter(type => typeof type === 'string' && type.length);
+            
             if (projectData.projectType === 'field_recon') {
                 projectData.frTypes = this.form.frTypes.filter(type => typeof type === 'string' && type.length);
                 if (isNaN(Date.parse(projectData.nhEventEnd))) {
                     projectData.nhEventEnd = new Date(projectData.nhEventStart);
                 }
             }
+            else {
+                let fields = ["nhEvent", "nhLocation", "nhLongitude", "nhLatitude"];
+                let result = fields.every((field) => {return typeof projectData[field] === 'string' && projectData[field].length})
+                let checkDate = isNaN(Date.parse(projectData.nhEventStart))
+                console.log(checkDate)
+                if (!result || checkDate) {
+                    fields.forEach((field) => projectData[field] = '')
+                    projectData.nhEventStart = ''
+                    projectData.nhEventEnd = ''
+                }
+                else {
+                    if (isNaN(Date.parse(projectData.nhEventEnd))) {
+                        projectData.nhEventEnd = new Date(projectData.nhEventStart);
+                    }
+                }
+            }
+            
         }
+        console.log(projectData)
```

Comment:
Remove comment

Reply:
removed all console.log statements I was using for testing